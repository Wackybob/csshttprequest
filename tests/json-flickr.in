{
		"title": "Recent Uploads tagged favorite",
		"link": "http://www.flickr.com/photos/tags/favorite/",
		"description": "",
		"modified": "2008-12-23T01:39:21Z",
		"generator": "http://www.flickr.com/",
		"items": [
	   {
			"title": "nyhavn",
			"link": "http://www.flickr.com/photos/mcm/3129802228/",
			"media": {"m":"http://farm4.static.flickr.com/3130/3129802228_4509c87972_m.jpg"},
			"date_taken": "2008-12-05T22:13:31-08:00",
			"description": "<p><a href=\"http://www.flickr.com/people/mcm/\">mcmCPH<\/a> posted a photo:<\/p> <p><a href=\"http://www.flickr.com/photos/mcm/3129802228/\" title=\"nyhavn\"><img src=\"http://farm4.static.flickr.com/3130/3129802228_4509c87972_m.jpg\" width=\"240\" height=\"180\" alt=\"nyhavn\" /><\/a><\/p> ",
			"published": "2008-12-23T01:39:21Z",
			"author": "nobody@flickr.com (mcmCPH)",
			"author_id": "47341069@N00",
			"tags": "favorite copenhagen denmark nyhavn"
	   },
	   {
			"title": "denmark - me and malcolm ice skating in copenhagen",
			"link": "http://www.flickr.com/photos/mcm/3129782702/",
			"media": {"m":"http://farm4.static.flickr.com/3229/3129782702_4da69b2b74_m.jpg"},
			"date_taken": "2008-09-29T11:31:29-08:00",
			"description": "<p><a href=\"http://www.flickr.com/people/mcm/\">mcmCPH<\/a> posted a photo:<\/p> <p><a href=\"http://www.flickr.com/photos/mcm/3129782702/\" title=\"denmark - me and malcolm ice skating in copenhagen\"><img src=\"http://farm4.static.flickr.com/3229/3129782702_4da69b2b74_m.jpg\" width=\"181\" height=\"240\" alt=\"denmark - me and malcolm ice skating in copenhagen\" /><\/a><\/p> ",
			"published": "2008-12-23T01:29:55Z",
			"author": "nobody@flickr.com (mcmCPH)",
			"author_id": "47341069@N00",
			"tags": "favorite copenhagen denmark michael malcolm frederiksberg"
	   },
	   {
			"title": "farewells2007",
			"link": "http://www.flickr.com/photos/mcm/3128878401/",
			"media": {"m":"http://farm4.static.flickr.com/3205/3128878401_bcdd01d3c7_m.jpg"},
			"date_taken": "2008-12-05T22:15:17-08:00",
			"description": "<p><a href=\"http://www.flickr.com/people/mcm/\">mcmCPH<\/a> posted a photo:<\/p> <p><a href=\"http://www.flickr.com/photos/mcm/3128878401/\" title=\"farewells2007\"><img src=\"http://farm4.static.flickr.com/3205/3128878401_bcdd01d3c7_m.jpg\" width=\"180\" height=\"240\" alt=\"farewells2007\" /><\/a><\/p> <p>a collage of farewells from when we left DK, July 2007<\/p>",
			"published": "2008-12-23T00:52:17Z",
			"author": "nobody@flickr.com (mcmCPH)",
			"author_id": "47341069@N00",
			"tags": "friends favorite denmark"
	   },
	   {
			"title": "Choices.",
			"link": "http://www.flickr.com/photos/marvelous_nature/3129615474/",
			"media": {"m":"http://farm4.static.flickr.com/3120/3129615474_d9f7a3d892_m.jpg"},
			"date_taken": "2008-12-22T17:04:41-08:00",
			"description": "<p><a href=\"http://www.flickr.com/people/marvelous_nature/\">marvelous_nature<\/a> posted a photo:<\/p> <p><a href=\"http://www.flickr.com/photos/marvelous_nature/3129615474/\" title=\"Choices.\"><img src=\"http://farm4.static.flickr.com/3120/3129615474_d9f7a3d892_m.jpg\" width=\"240\" height=\"180\" alt=\"Choices.\" /><\/a><\/p> <p>&quot;Buddy the Elf! What\'s your favorite color?&quot; <br /> <br /> Choose one! :)<\/p>",
			"published": "2008-12-23T00:04:41Z",
			"author": "nobody@flickr.com (marvelous_nature)",
			"author_id": "27243539@N06",
			"tags": "favorite color green thread sewing shades choices"
	   },
	   {
			"title": "denmark - first day at bornehave on christiania - susan and malcolm",
			"link": "http://www.flickr.com/photos/mcm/3128743785/",
			"media": {"m":"http://farm4.static.flickr.com/3255/3128743785_281a363c36_m.jpg"},
			"date_taken": "2008-12-22T15:42:38-08:00",
			"description": "<p><a href=\"http://www.flickr.com/people/mcm/\">mcmCPH<\/a> posted a photo:<\/p> <p><a href=\"http://www.flickr.com/photos/mcm/3128743785/\" title=\"denmark - first day at bornehave on christiania - susan and malcolm\"><img src=\"http://farm4.static.flickr.com/3255/3128743785_281a363c36_m.jpg\" width=\"226\" height=\"240\" alt=\"denmark - first day at bornehave on christiania - susan and malcolm\" /><\/a><\/p> ",
			"published": "2008-12-22T23:42:38Z",
			"author": "nobody@flickr.com (mcmCPH)",
			"author_id": "47341069@N00",
			"tags": "favorite denmark malcolm susan christiania børnehave birkerød"
	   },
	   {
			"title": "Panorama desde la Alberca Caesar\'s",
			"link": "http://www.flickr.com/photos/rafachapa/3128676265/",
			"media": {"m":"http://farm4.static.flickr.com/3206/3128676265_c413958e81_m.jpg"},
			"date_taken": "2008-10-29T08:49:24-08:00",
			"description": "<p><a href=\"http://www.flickr.com/people/rafachapa/\">rafachapa<\/a> posted a photo:<\/p> <p><a href=\"http://www.flickr.com/photos/rafachapa/3128676265/\" title=\"Panorama desde la Alberca Caesar\'s\"><img src=\"http://farm4.static.flickr.com/3206/3128676265_c413958e81_m.jpg\" width=\"159\" height=\"240\" alt=\"Panorama desde la Alberca Caesar\'s\" /><\/a><\/p> ",
			"published": "2008-12-22T23:08:47Z",
			"author": "nobody@flickr.com (rafachapa)",
			"author_id": "8960796@N03",
			"tags": "trees panorama sculpture horse favorite usa water caballo hotel agua américa arboles lasvegas nevada esculturas nv strip animales caesarspalace 2008 estadosunidos eeuu"
	   },
	   {
			"title": "Panorama Lobby Caesar\'s de Noche - 2",
			"link": "http://www.flickr.com/photos/rafachapa/3129463284/",
			"media": {"m":"http://farm4.static.flickr.com/3262/3129463284_e707c40218_m.jpg"},
			"date_taken": "2008-10-28T23:16:57-08:00",
			"description": "<p><a href=\"http://www.flickr.com/people/rafachapa/\">rafachapa<\/a> posted a photo:<\/p> <p><a href=\"http://www.flickr.com/photos/rafachapa/3129463284/\" title=\"Panorama Lobby Caesar\'s de Noche - 2\"><img src=\"http://farm4.static.flickr.com/3262/3129463284_e707c40218_m.jpg\" width=\"240\" height=\"93\" alt=\"Panorama Lobby Caesar\'s de Noche - 2\" /><\/a><\/p> ",
			"published": "2008-12-22T22:48:06Z",
			"author": "nobody@flickr.com (rafachapa)",
			"author_id": "8960796@N03",
			"tags": "wallpaper panorama sculpture favorite usa detalle detail restaurant hotel américa lasvegas nevada decoration esculturas nv strip caesarspalace fountains 2008 fuentes slotmachine estadosunidos onearmbandit eeuu decoración maquinitas tragamonedas"
	   },
	   {
			"title": "Panorama Venetian Mirage 08",
			"link": "http://www.flickr.com/photos/rafachapa/3129455268/",
			"media": {"m":"http://farm4.static.flickr.com/3198/3129455268_276c8bcfee_m.jpg"},
			"date_taken": "2008-10-28T22:07:31-08:00",
			"description": "<p><a href=\"http://www.flickr.com/people/rafachapa/\">rafachapa<\/a> posted a photo:<\/p> <p><a href=\"http://www.flickr.com/photos/rafachapa/3129455268/\" title=\"Panorama Venetian Mirage 08\"><img src=\"http://farm4.static.flickr.com/3198/3129455268_276c8bcfee_m.jpg\" width=\"240\" height=\"104\" alt=\"Panorama Venetian Mirage 08\" /><\/a><\/p> ",
			"published": "2008-12-22T22:44:38Z",
			"author": "nobody@flickr.com (rafachapa)",
			"author_id": "8960796@N03",
			"tags": "longexposure trees wallpaper panorama favorite usa night ads hotel noche américa arboles lasvegas nevada movimiento nv strip mirage 2008 thebeatles estadosunidos anuncios thevenetian eeuu largaexposición"
	   },
	   {
			"title": "The Strip at Night",
			"link": "http://www.flickr.com/photos/rafachapa/3128614027/",
			"media": {"m":"http://farm4.static.flickr.com/3133/3128614027_b725fb3f1f_m.jpg"},
			"date_taken": "2008-09-15T23:15:30-08:00",
			"description": "<p><a href=\"http://www.flickr.com/people/rafachapa/\">rafachapa<\/a> posted a photo:<\/p> <p><a href=\"http://www.flickr.com/photos/rafachapa/3128614027/\" title=\"The Strip at Night\"><img src=\"http://farm4.static.flickr.com/3133/3128614027_b725fb3f1f_m.jpg\" width=\"240\" height=\"180\" alt=\"The Strip at Night\" /><\/a><\/p> ",
			"published": "2008-12-22T22:39:34Z",
			"author": "nobody@flickr.com (rafachapa)",
			"author_id": "8960796@N03",
			"tags": "wallpaper favorite usa water night ads hotel noche agua américa lasvegas nevada nv strip bellagio fountains planethollywood ballys 2008 citycenter fuentes parislasvegas estadosunidos anuncios eeuu"
	   },
	   {
			"title": "Path to Wynn",
			"link": "http://www.flickr.com/photos/rafachapa/3128552637/",
			"media": {"m":"http://farm4.static.flickr.com/3292/3128552637_c52084a68a_m.jpg"},
			"date_taken": "2008-09-15T16:37:38-08:00",
			"description": "<p><a href=\"http://www.flickr.com/people/rafachapa/\">rafachapa<\/a> posted a photo:<\/p> <p><a href=\"http://www.flickr.com/photos/rafachapa/3128552637/\" title=\"Path to Wynn\"><img src=\"http://farm4.static.flickr.com/3292/3128552637_c52084a68a_m.jpg\" width=\"240\" height=\"180\" alt=\"Path to Wynn\" /><\/a><\/p> ",
			"published": "2008-12-22T22:12:02Z",
			"author": "nobody@flickr.com (rafachapa)",
			"author_id": "8960796@N03",
			"tags": "trees wallpaper favorite usa ads hotel américa arboles treasureisland lasvegas nevada nv strip wynn 2008 encore estadosunidos anuncios fashionshowmall eeuu favorito"
	   },
	   {
			"title": "X-MAS FAVE",
			"link": "http://www.flickr.com/photos/mundoperfeito/3129365080/",
			"media": {"m":"http://farm4.static.flickr.com/3252/3129365080_1e36a7c459_m.jpg"},
			"date_taken": "2008-12-22T20:04:03-08:00",
			"description": "<p><a href=\"http://www.flickr.com/people/mundoperfeito/\">MundoPerfeito<\/a> posted a photo:<\/p> <p><a href=\"http://www.flickr.com/photos/mundoperfeito/3129365080/\" title=\"X-MAS FAVE\"><img src=\"http://farm4.static.flickr.com/3252/3129365080_1e36a7c459_m.jpg\" width=\"240\" height=\"240\" alt=\"X-MAS FAVE\" /><\/a><\/p> <p>1. <a href=\"http://flickr.com/photos/14422482@N00/3123343058/\">Merry X-Mas!<\/a>, 2. <a href=\"http://flickr.com/photos/22882495@N00/3125022609/\">Cupcakes<\/a>, 3. <a href=\"http://flickr.com/photos/66942581@N00/3128342871/\">wishing on a star.<\/a>, 4. <a href=\"http://flickr.com/photos/46973832@N00/3118385296/\">*muse*<\/a>, 5. <a href=\"http://flickr.com/photos/45562894@N00/3116650022/\">Untitled<\/a>, 6. <a href=\"http://flickr.com/photos/45562894@N00/3114368952/\">Untitled<\/a>, 7. <a href=\"http://flickr.com/photos/42933826@N00/3113362205/\">snowflakes<\/a>, 8. <a href=\"http://flickr.com/photos/53611153@N00/3112932019/\">bokeh berries<\/a>, 9. <a href=\"http://flickr.com/photos/85824446@N00/3112493885/\">Untitled<\/a>, 10. <a href=\"http://flickr.com/photos/66942581@N00/3077470137/\">untold stories<\/a>, 11. <a href=\"http://flickr.com/photos/12079881@N00/3097253336/\">i see christmas bokeh everywhere...<\/a>, 12. <a href=\"http://flickr.com/photos/38242913@N00/3110568000/\">Xmas Birds<\/a>, 13. <a href=\"http://flickr.com/photos/19055148@N00/3110320492/\">Reaching for a star 2008!<\/a>, 14. <a href=\"http://flickr.com/photos/65086288@N00/3107683961/\">Anjos<\/a>, 15. <a href=\"http://flickr.com/photos/94161900@N00/3105865650/\">Sangue di Dionisio<\/a>, 16. <a href=\"http://flickr.com/photos/81877607@N00/3101178934/\">self-portrait on a christmas ball<\/a><br /> <br /> Created with <a href=\"http://bighugelabs.com/flickr/\">fd\'s Flickr Toys<\/a>.<\/p>",
			"published": "2008-12-22T22:04:03Z",
			"author": "nobody@flickr.com (MundoPerfeito)",
			"author_id": "56153257@N00",
			"tags": "favorite fdsflickrtoys mosaic mosaico faves favoritos"
	   },
	   {
			"title": "Wynn and Encore from the Bay - 3",
			"link": "http://www.flickr.com/photos/rafachapa/3128528607/",
			"media": {"m":"http://farm4.static.flickr.com/3091/3128528607_50633094ba_m.jpg"},
			"date_taken": "2008-09-15T16:35:00-08:00",
			"description": "<p><a href=\"http://www.flickr.com/people/rafachapa/\">rafachapa<\/a> posted a photo:<\/p> <p><a href=\"http://www.flickr.com/photos/rafachapa/3128528607/\" title=\"Wynn and Encore from the Bay - 3\"><img src=\"http://farm4.static.flickr.com/3091/3128528607_50633094ba_m.jpg\" width=\"240\" height=\"180\" alt=\"Wynn and Encore from the Bay - 3\" /><\/a><\/p> ",
			"published": "2008-12-22T22:00:54Z",
			"author": "nobody@flickr.com (rafachapa)",
			"author_id": "8960796@N03",
			"tags": "trees wallpaper favorite usa water ads hotel agua américa arboles treasureisland lasvegas nevada nv strip wynn 2008 estadosunidos anuncios eeuu"
	   },
	   {
			"title": "Wynn and Encore from the Bay - 2",
			"link": "http://www.flickr.com/photos/rafachapa/3129353258/",
			"media": {"m":"http://farm4.static.flickr.com/3178/3129353258_e3171fa245_m.jpg"},
			"date_taken": "2008-09-15T16:34:37-08:00",
			"description": "<p><a href=\"http://www.flickr.com/people/rafachapa/\">rafachapa<\/a> posted a photo:<\/p> <p><a href=\"http://www.flickr.com/photos/rafachapa/3129353258/\" title=\"Wynn and Encore from the Bay - 2\"><img src=\"http://farm4.static.flickr.com/3178/3129353258_e3171fa245_m.jpg\" width=\"240\" height=\"180\" alt=\"Wynn and Encore from the Bay - 2\" /><\/a><\/p> ",
			"published": "2008-12-22T21:58:49Z",
			"author": "nobody@flickr.com (rafachapa)",
			"author_id": "8960796@N03",
			"tags": "trees wallpaper favorite usa water ads hotel agua américa arboles treasureisland lasvegas nevada nv strip wynn 2008 estadosunidos anuncios eeuu"
	   },
	   {
			"title": "Net Worth",
			"link": "http://www.flickr.com/photos/rafachapa/3129347366/",
			"media": {"m":"http://farm4.static.flickr.com/3230/3129347366_6eaafd1a4e_m.jpg"},
			"date_taken": "2008-09-15T16:33:48-08:00",
			"description": "<p><a href=\"http://www.flickr.com/people/rafachapa/\">rafachapa<\/a> posted a photo:<\/p> <p><a href=\"http://www.flickr.com/photos/rafachapa/3129347366/\" title=\"Net Worth\"><img src=\"http://farm4.static.flickr.com/3230/3129347366_6eaafd1a4e_m.jpg\" width=\"240\" height=\"180\" alt=\"Net Worth\" /><\/a><\/p> ",
			"published": "2008-12-22T21:56:17Z",
			"author": "nobody@flickr.com (rafachapa)",
			"author_id": "8960796@N03",
			"tags": "trees wallpaper favorite usa detalle detail water hotel agua américa arboles treasureisland lasvegas nevada nv strip wynn 2008 estadosunidos thevenetian eeuu"
	   },
	   {
			"title": "Pirate Boat - 1",
			"link": "http://www.flickr.com/photos/rafachapa/3129334974/",
			"media": {"m":"http://farm4.static.flickr.com/3263/3129334974_d72be7a466_m.jpg"},
			"date_taken": "2008-09-15T16:32:24-08:00",
			"description": "<p><a href=\"http://www.flickr.com/people/rafachapa/\">rafachapa<\/a> posted a photo:<\/p> <p><a href=\"http://www.flickr.com/photos/rafachapa/3129334974/\" title=\"Pirate Boat - 1\"><img src=\"http://farm4.static.flickr.com/3263/3129334974_d72be7a466_m.jpg\" width=\"180\" height=\"240\" alt=\"Pirate Boat - 1\" /><\/a><\/p> ",
			"published": "2008-12-22T21:50:55Z",
			"author": "nobody@flickr.com (rafachapa)",
			"author_id": "8960796@N03",
			"tags": "trees favorite usa water hotel boat agua américa arboles treasureisland lasvegas nevada nv strip wynn 2008 bote estadosunidos eeuu"
	   },
	   {
			"title": "Caesar\'s Choice - 4",
			"link": "http://www.flickr.com/photos/rafachapa/3128489659/",
			"media": {"m":"http://farm4.static.flickr.com/3266/3128489659_b5f68d0011_m.jpg"},
			"date_taken": "2008-09-15T12:34:57-08:00",
			"description": "<p><a href=\"http://www.flickr.com/people/rafachapa/\">rafachapa<\/a> posted a photo:<\/p> <p><a href=\"http://www.flickr.com/photos/rafachapa/3128489659/\" title=\"Caesar\'s Choice - 4\"><img src=\"http://farm4.static.flickr.com/3266/3128489659_b5f68d0011_m.jpg\" width=\"180\" height=\"240\" alt=\"Caesar\'s Choice - 4\" /><\/a><\/p> ",
			"published": "2008-12-22T21:43:13Z",
			"author": "nobody@flickr.com (rafachapa)",
			"author_id": "8960796@N03",
			"tags": "sculpture favorite usa ads hotel américa lasvegas nevada esculturas nv strip caesarspalace ballys 2008 parislasvegas estadosunidos anuncios eeuu construcciones"
	   },
	   {
			"title": "Wynn and Encore from the Bay - 1",
			"link": "http://www.flickr.com/photos/rafachapa/3129329092/",
			"media": {"m":"http://farm4.static.flickr.com/3207/3129329092_3f7f2e81f9_m.jpg"},
			"date_taken": "2008-09-15T16:32:07-08:00",
			"description": "<p><a href=\"http://www.flickr.com/people/rafachapa/\">rafachapa<\/a> posted a photo:<\/p> <p><a href=\"http://www.flickr.com/photos/rafachapa/3129329092/\" title=\"Wynn and Encore from the Bay - 1\"><img src=\"http://farm4.static.flickr.com/3207/3129329092_3f7f2e81f9_m.jpg\" width=\"240\" height=\"180\" alt=\"Wynn and Encore from the Bay - 1\" /><\/a><\/p> ",
			"published": "2008-12-22T21:48:07Z",
			"author": "nobody@flickr.com (rafachapa)",
			"author_id": "8960796@N03",
			"tags": "trees wallpaper favorite usa water hotel boat store agua américa arboles treasureisland lasvegas nevada nv tienda strip wynn 2008 bote estadosunidos eeuu favorito"
	   },
	   {
			"title": "Independencia de México - 1",
			"link": "http://www.flickr.com/photos/rafachapa/3128472897/",
			"media": {"m":"http://farm4.static.flickr.com/3128/3128472897_d068b57ee5_m.jpg"},
			"date_taken": "2008-09-15T12:32:19-08:00",
			"description": "<p><a href=\"http://www.flickr.com/people/rafachapa/\">rafachapa<\/a> posted a photo:<\/p> <p><a href=\"http://www.flickr.com/photos/rafachapa/3128472897/\" title=\"Independencia de México - 1\"><img src=\"http://farm4.static.flickr.com/3128/3128472897_d068b57ee5_m.jpg\" width=\"240\" height=\"180\" alt=\"Independencia de México - 1\" /><\/a><\/p> ",
			"published": "2008-12-22T21:35:40Z",
			"author": "nobody@flickr.com (rafachapa)",
			"author_id": "8960796@N03",
			"tags": "trees wallpaper sculpture favorite usa detalle detail water ads hotel agua américa arboles lasvegas nevada esculturas nv strip carro caesarspalace fountains 2008 fuentes estadosunidos anuncios eeuu construcciones favorito"
	   },
	   {
			"title": "Shadow at Caesar\'s Palace - 1",
			"link": "http://www.flickr.com/photos/rafachapa/3128462155/",
			"media": {"m":"http://farm4.static.flickr.com/3079/3128462155_7fffba38a2_m.jpg"},
			"date_taken": "2008-09-15T12:12:00-08:00",
			"description": "<p><a href=\"http://www.flickr.com/people/rafachapa/\">rafachapa<\/a> posted a photo:<\/p> <p><a href=\"http://www.flickr.com/photos/rafachapa/3128462155/\" title=\"Shadow at Caesar\'s Palace - 1\"><img src=\"http://farm4.static.flickr.com/3079/3128462155_7fffba38a2_m.jpg\" width=\"240\" height=\"180\" alt=\"Shadow at Caesar\'s Palace - 1\" /><\/a><\/p> ",
			"published": "2008-12-22T21:30:50Z",
			"author": "nobody@flickr.com (rafachapa)",
			"author_id": "8960796@N03",
			"tags": "wallpaper favorite usa bar ads restaurant hotel américa shadows lasvegas nevada nv strip caesarspalace 2008 sombras unfocus estadosunidos anuncios desenfocada eeuu"
	   },
	   {
			"title": "By the Pool - 1",
			"link": "http://www.flickr.com/photos/rafachapa/3128423615/",
			"media": {"m":"http://farm4.static.flickr.com/3228/3128423615_05d9edb75d_m.jpg"},
			"date_taken": "2008-09-15T12:00:10-08:00",
			"description": "<p><a href=\"http://www.flickr.com/people/rafachapa/\">rafachapa<\/a> posted a photo:<\/p> <p><a href=\"http://www.flickr.com/photos/rafachapa/3128423615/\" title=\"By the Pool - 1\"><img src=\"http://farm4.static.flickr.com/3228/3128423615_05d9edb75d_m.jpg\" width=\"240\" height=\"180\" alt=\"By the Pool - 1\" /><\/a><\/p> ",
			"published": "2008-12-22T21:13:47Z",
			"author": "nobody@flickr.com (rafachapa)",
			"author_id": "8960796@N03",
			"tags": "2008 nevada américa nv estadosunidos lasvegas eeuu usa caesarspalace strip esculturas sculpture hotel animales horse caballo arboles trees construcciones wallpaper favorite"
	   }
        ]
}
