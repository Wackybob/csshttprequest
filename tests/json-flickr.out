#c0{background:url(data:,%7B%0A%09%09%22title%22%3A%20%22Recent%20Uploads%20tagged%20favorite%22%2C%0A%09%09%22link%22%3A%20%22http%3A//www.flickr.com/photos/tags/favorite/%22%2C%0A%09%09%22description%22%3A%20%22%22%2C%0A%09%09%22modified%22%3A%20%222008-12-23T01%3A39%3A21Z%22%2C%0A%09%09%22generator%22%3A%20%22http%3A//www.flickr.com/%22%2C%0A%09%09%22items%22%3A%20%5B%0A%09%20%20%20%7B%0A%09%09%09%22title%22%3A%20%22nyhavn%22%2C%0A%09%09%09%22link%22%3A%20%22http%3A//www.flickr.com/photos/mcm/3129802228/%22%2C%0A%09%09%09%22media%22%3A%20%7B%22m%22%3A%22http%3A//farm4.static.flickr.com/3130/3129802228_4509c87972_m.jpg%22%7D%2C%0A%09%09%09%22date_taken%22%3A%20%222008-12-05T22%3A13%3A31-08%3A00%22%2C%0A%09%09%09%22description%22%3A%20%22%3Cp%3E%3Ca%20href%3D%5C%22http%3A//www.flickr.com/people/mcm/%5C%22%3EmcmCPH%3C%5C/a%3E%20posted%20a%20photo%3A%3C%5C/p%3E%20%3Cp%3E%3Ca%20href%3D%5C%22http%3A//www.flickr.com/photos/mcm/3129802228/%5C%22%20title%3D%5C%22nyhavn%5C%22%3E%3Cimg%20src%3D%5C%22http%3A//farm4.static.flickr.com/3130/3129802228_4509c87972_m.jpg%5C%22%20width%3D%5C%22240%5C%22%20height%3D%5C%22180%5C%22%20alt%3D%5C%22nyhavn%5C%22%20/%3E%3C%5C/a%3E%3C%5C/p%3E%20%22%2C%0A%09%09%09%22published%22%3A%20%222008-12-23T01%3A39%3A21Z%22%2C%0A%09%09%09%22author%22%3A%20%22nobody%40flickr.com%20%28mcmCPH%29%22%2C%0A%09%09%09%22author_id%22%3A%20%2247341069%40N00%22%2C%0A%09%09%09%22tags%22%3A%20%22favorite%20copenhagen%20denmark%20nyhavn%22%0A%09%20%20%20%7D%2C%0A%09%20%20%20%7B%0A%09%09%09%22title%22%3A%20%22denmark%20-%20me%20and%20malcolm%20ice%20skating%20in%20copenhagen%22%2C%0A%09%09%09%22link%22%3A%20%22http%3A//www.flickr.com/photos/mcm/3129782702/%22%2C%0A%09%09%09%22media%22%3A%20%7B%22m%22%3A%22http%3A//farm4.static.flickr.com/3229/3129782702_4da69b2b74_m.jpg%22%7D%2C%0A%09%09%09%22date_taken%22%3A%20%222008-09-29T11%3A31%3A29-08%3A00%22%2C%0A%09%09%09%22description%22%3A%20%22%3Cp%3E%3Ca%20href%3D%5C%22http%3A//www.flickr.com/people/mcm/%5C%22%3EmcmCPH%3C%5C/a%3E%20posted);}
#c1{background:url(data:,%20a%20photo%3A%3C%5C/p%3E%20%3Cp%3E%3Ca%20href%3D%5C%22http%3A//www.flickr.com/photos/mcm/3129782702/%5C%22%20title%3D%5C%22denmark%20-%20me%20and%20malcolm%20ice%20skating%20in%20copenhagen%5C%22%3E%3Cimg%20src%3D%5C%22http%3A//farm4.static.flickr.com/3229/3129782702_4da69b2b74_m.jpg%5C%22%20width%3D%5C%22181%5C%22%20height%3D%5C%22240%5C%22%20alt%3D%5C%22denmark%20-%20me%20and%20malcolm%20ice%20skating%20in%20copenhagen%5C%22%20/%3E%3C%5C/a%3E%3C%5C/p%3E%20%22%2C%0A%09%09%09%22published%22%3A%20%222008-12-23T01%3A29%3A55Z%22%2C%0A%09%09%09%22author%22%3A%20%22nobody%40flickr.com%20%28mcmCPH%29%22%2C%0A%09%09%09%22author_id%22%3A%20%2247341069%40N00%22%2C%0A%09%09%09%22tags%22%3A%20%22favorite%20copenhagen%20denmark%20michael%20malcolm%20frederiksberg%22%0A%09%20%20%20%7D%2C%0A%09%20%20%20%7B%0A%09%09%09%22title%22%3A%20%22farewells2007%22%2C%0A%09%09%09%22link%22%3A%20%22http%3A//www.flickr.com/photos/mcm/3128878401/%22%2C%0A%09%09%09%22media%22%3A%20%7B%22m%22%3A%22http%3A//farm4.static.flickr.com/3205/3128878401_bcdd01d3c7_m.jpg%22%7D%2C%0A%09%09%09%22date_taken%22%3A%20%222008-12-05T22%3A15%3A17-08%3A00%22%2C%0A%09%09%09%22description%22%3A%20%22%3Cp%3E%3Ca%20href%3D%5C%22http%3A//www.flickr.com/people/mcm/%5C%22%3EmcmCPH%3C%5C/a%3E%20posted%20a%20photo%3A%3C%5C/p%3E%20%3Cp%3E%3Ca%20href%3D%5C%22http%3A//www.flickr.com/photos/mcm/3128878401/%5C%22%20title%3D%5C%22farewells2007%5C%22%3E%3Cimg%20src%3D%5C%22http%3A//farm4.static.flickr.com/3205/3128878401_bcdd01d3c7_m.jpg%5C%22%20width%3D%5C%22180%5C%22%20height%3D%5C%22240%5C%22%20alt%3D%5C%22farewells2007%5C%22%20/%3E%3C%5C/a%3E%3C%5C/p%3E%20%3Cp%3Ea%20collage%20of%20farewells%20from%20when%20we%20left%20DK%2C%20July%202007%3C%5C/p%3E%22%2C%0A%09%09%09%22published%22%3A%20%222008-12-23T00%3A52%3A17Z%22%2C%0A%09%09%09%22author%22%3A%20%22nobody%40flickr.com%20%28mcmCPH%29%22%2C%0A%09%09%09%22author_id%22%3A%20%2247341069%40N00%22%2C%0A%09%09%09%22tags%22%3A%20%22friends%20favorite%20denmark%22%0A%09%20%20);}
#c2{background:url(data:,%20%7D%2C%0A);}
